src/character.cpp

```diff
@@ -71,7 +71,12 @@ void Character::mod_stat( const std::string &stat, int modifier )
     } else if( stat == "healthy" ) {
         mod_healthy( modifier );
     } else if( stat == "healthy_mod" ) {
-        mod_healthy_mod( modifier );
+        debugmsg("mod_stat should not be used for healthy_mod");
```

Comment:
If you're sure `mod_stat` shouldn't be used to mod `healthy_mod`, just remove the entire section. `Creature::mod_stat` (below) will debugmsg if it gets an unhandled stat.
Otherwise just let it have the -200/200 mods, without the debugmsg.